pep-0001.txt

```diff
@@ -777,23 +777,11 @@ References and Footnotes
    for retrieving older revisions, and can also be browsed via HTTP here:
    https://github.com/python/peps
 
-.. [2] PEP 2, Procedure for Adding New Modules
-   (http://www.python.org/dev/peps/pep-0002)
```

Comment:
The previous system was a bit weird with its footnotes instead of inline links, but one good point was that you could see the linked PEP title without following the link.

Since we are in the PEP repo (with the data on hand without network requests needed), could the code be adapted so that the PEP number and title is used as HTML `title` attribute on the generated links?

Reply:
Fantastic suggestion! That would be _really_ nice and something I've been wanting for a while myself. (To note, with manual footnotes you still have to click the link to go to the footnote to see it of course, it just doesn't generate a HTTP request)

Reply:
Will have a look, seems pretty easy.